webpack.config.js

```diff
@@ -23,6 +23,16 @@ module.exports = {
         test: /\.(png|svg|jpg|jpeg|gif)$/i,
         type: 'asset/resource',
       },
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env'],
+          },
+        },
```

Comment:
- [ ] You have done great so far but there are still some issues you need to resolve. :+1: :+1: As pointed out by the previous reviewer, when I ran `npm run start`, I got errors in the browser and terminal as shown in the screenshot below, I'll suggest you go back to the lesson on setting up Webpack and try using it to troubleshoot the error. :wave: :+1: :smile: 

- - Screenshot:
 
![Screenshot from 2023-07-22 19-03-32](https://github.com/AndresEspi/toDoList/assets/107632780/5f1d48ed-8245-4542-b1cf-a131ca232064)